src/Prometheus/Storage/Redis.php

```diff
@@ -92,12 +92,29 @@ public static function setPrefix(string $prefix): void
     }
 
     /**
+     * Atomically removes all previously stored data from redis
+     *
      * @throws StorageException
      */
     public function flushRedis(): void
```

Comment:
I think it makes sense to leave it public, to help people clean their storages if needed.
But I would make it a method in the interface and maybe rename to something like `wipeStorage`.

Reply:
Make sense, i will make that change. Maybe more consistent name with current implementation will be `flushStorage`? I am terrible at naming things, so i will take any advice

Reply:
Please check the latest version. I added the proposed change to Adapter interface and marked old methods as deprecated

Reply:
`flushStorage` is also good indeed. thanks!

Reply:
Little late but fwiw also agree with the direction of these changes. I think it should be kept public in case clearing metrics is part of somebody's workflow, but absolutely should not clear the entire Redis.